Property_map/include/CGAL/property_map.h

```diff
@@ -173,7 +167,7 @@ struct First_of_pair_property_map
 {
   typedef Pair key_type; ///< typedef to `Pair`
   typedef typename Pair::first_type value_type; ///< typedef to `Pair::first_type`
-  typedef const value_type& reference; ///< typedef to `value_type&`
+  typedef value_type& reference; ///< typedef to `value_type&`
```

Comment:
`key_type` and `reference` should be related.

Using `boost::copy_cv`, they could copy the constness of `Pair`:
```
typedef Pair& key_type;
typedef typename Pair::first_type value_type;
typedef typename boost::copy_cv<typename Pair::first_type, Pair>::type& reference;
```